torch/utils/check_import.py

```diff
@@ -0,0 +1,146 @@
+# This script tries to figure out the reason of
+# `ImportError` on Windows.
+# Run it with `python check_import.py`.
+
+import ctypes
+import glob
+import os
+import sys
+import subprocess
+
+PY3 = sys.version_info >= (3, 0)
+
+TORCH_ROOT = os.path.dirname(os.path.dirname(__file__))
+PY_DLL_PATH = os.path.join(os.path.dirname(sys.executable), 'Library\\bin')
```

Comment:
Isn't it more portable to write `'Library', 'bin'` here?

Reply:
I guess, not that portability matters, since this is a Windows specific path :o)